cranelift/filetests/filetests/isa/x64/urem.clif

```diff
@@ -44,8 +45,9 @@ block0(v0: i32, v1: i32):
 ;   pushq   %rbp
 ;   movq    %rsp, %rbp
 ; block0:
+;   movl    $0, %r11d
 ;   movq    %rdi, %rax
-;   movl    $0, %edx
+;   movq    %r11, %rdx
```

Comment:
It looks like it's the result of a slight overapproximation of the liverange of the rdx input and rdx output: the input is supposed to be live up to the early-point and output live starting at the late-point, but for I think fuzzbug-related reasons that I don't fully recall, I had to make fixed defs live starting at the early point. The conflict is resolved by putting the input in r11 here but doing a last-minute move. I can dig into it further in regalloc2 but it will take some deep thought; not a super-easy fix. In any case the operands express a more flexible problem so we should be able to solve it locally (within regalloc) now...